frontend/src/components/articles/articleListing.tsx

```diff
@@ -74,25 +74,52 @@ const ArticleListing: FunctionalComponent<Props> = ({ section }) => {
     let singleArticleLink = `/section/${formatUrl(article.tag)}/${article.tag_meta ? formatUrl(article.tag_meta) + '/' : ''}${formatUrl(article.title)}/${article.id}`
     return (
       <div class={style.mobileHomeArticleContainer}>
-        <div class={style.mobileTagContainer}>
-          <p class={style.mobileTag}>{article.tag}</p>
-          <div class={style.share}>
+
+        <div class={style.content}>
+          <div class={style.mobileTagContainer}>
+            <h1 class={style.mobileTag}>{article.tag}</h1>
           </div>
-        </div>
 
-        <div >
-          <Link href={singleArticleLink}>
-            <img class={style.mainImage} src={article.img_src} />
+          <Link
```

Comment:
For readability purposes, if you can replace this href string with the variable singleArticleLink. I had updated that but seems to have gotten deleted when the changes were merged.